packages/code-du-travail-frontend/server.js

```diff
@@ -3,18 +3,50 @@ const express = require("express");
 const next = require("next");
 const routes = require("./routes");
 
+/**
+ * this env variable is use to target developpement / staging deployement
+ * in order to block indexing bot using a x-robot-header and an appropriate robots.txt
+ */
+const LIVEPROD = process.env.ENVIRONMENT === "production";
```

Comment:
Don't we use the NODE_ENV variable ? I thought it was some sort of convention

Reply:
NODE_ENV is generally used to build a "production" bundle
we use production bundle for  dev / staging environment
What I need is to know if the env is the production site (aka codedutravail.num.social.gouv.fr) or any other deployement (aka codedutravail-dev.num.social.gouv.fr, xxxxxx.code-du-travail-numerique.dev.factory.social.gouv.fr) to handle the robots.txt accordingly